src/Response.php

```diff
@@ -80,7 +80,7 @@ class Response implements ResponseInterface
     private $statusCode = 200;
 
     /**
-     * @param int                                  $status  Status code
+     * @param int|string                           $status  Status code
```

Comment:
note that I have also changed the typehint..

Reply:
please revert as I don't want to support string explicitly. it only happens to work for bc.

Reply:
done